libraries/chatSpaces/README.md

```diff
@@ -0,0 +1,71 @@
+# Chat Spaces
+
+Chat spaces are integrations that connect Bott to external chat platforms like
+Discord, Slack, or other messaging services.
+
+## Requirements
+
+All chat space integrations must adhere to the following standards to ensure
+proper operation with the Bott system.
+
+### User Mention Format
+
+When converting messages from external platforms to Bott events, user mentions
+must be formatted as:
+
+```
+@<personaId>
+```
+
+Where `personaId` is the unique identifier for the user's persona in the
+specific space.
+
+#### Examples
+
+- Discord mention `<@123456789>` should become `@<123456789>` in the BottEvent
+  content
+- Slack mention `<@U12345678>` should become `@<U12345678>` in the BottEvent
+  content
+
+#### Persona Management
```

Comment:
Slightly redundant with the "checklist" below. I'd merge.

Also we should link to this in the main README.md in the place where we list the supported integrations. possibly in CONTRIBUTING.md as well.